component_tabs.yaml

```diff
@@ -0,0 +1,43 @@
+# Settings: accessibility, desing_guidelines
+
+patterns/accordion:
+  accessibility: True
+patterns/breadcrumbs:
+  accessibility: True
+patterns/buttons:
+  accessibility: True
+patterns/card:
+  accessibility: True
+patterns/contextual-menu:
```

Comment:
Should this one be empty? It renders incorrectly without title at all.

I don't know if it's supposed to have tabs or not. If not I guess it shouldn't be in the file at all?

Reply:
We haven't updated the accessibility docs for this page, so currently there is a short paragraph:

"When using the `p-contextual-menu__toggle` class on a `button` element, please ensure that the button label contains a trailing ellipsis `…`, e.g. "Take action&hellip;". This is a convention used to indicate that the button launches a dialog.

In cases where a contextual menu is shown on click, focus should be set within the menu element, using JavaScript."

It feels like overkill to have a tab for this, so I'm leaning towards putting the text back into the main file and having no tabs for this page until we have complete accessibility docs

Reply:
Sounds good.